hawkbit-core/src/main/java/org/eclipse/hawkbit/exception/SpServerError.java

```diff
@@ -164,15 +164,20 @@
     /**
      * 
      */
-    SP_ROLLOUT_ILLEGAL_STATE("hawkbit.server.error.rollout.illegalstate", "The rollout is currently in the wrong state for the current operation");
+    SP_ROLLOUT_ILLEGAL_STATE("hawkbit.server.error.rollout.illegalstate", "The rollout is in the wrong state for the requested operation"),
+
+    /**
+     *
+     */
+    SP_ROLLOUT_VERIFICATION_FAILED("hawkbit.server.error.rollout.verificationFailed", "The rollout configuration could not be verified successfully");
 
     private final String key;
     private final String message;
 
     /*
      * Repository side Error codes
      */
-    private SpServerError(final String errorKey, final String message) {
+    SpServerError(final String errorKey, final String message) {
```

Comment:
It is an enum, that has by default a private constructor. I removed the redundant modifier.